packages/bpk-component-page-indicator/src/NavButton.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2016 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/* @flow strict */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { withButtonAlignment, withRtlSupport } from '../../bpk-component-icon';
+import BpkButton from '../../bpk-component-button';
+import LeftArrowIcon from '../../bpk-component-icon/lg/chevron-left';
+import RightArrowIcon from '../../bpk-component-icon/lg/chevron-right';
+
+type Props = {
+  prev?: boolean,
```

Comment:
I hope I do not move the conversations back to what already was discussed and decided upon 😅  But still, I think `prev: boolean` is pretty confusing here even for internal Indicators API. Since we already have directions, we could re-use the keys, I think this is easier to justify:

```tsx
<NavButton direction={DIRECTIONS.PREV} onClick={() => onClick(e, currentIndex - 1, DIRECTIONS.PREV)} />
<Indicators />
<NavButton direction={DIRECTIONS.NEXT} onClick={() => onClick(e, currentIndex + 1, DIRECTIONS.NEXT)} />
```

Reply:
+1 and good catch, using booleans is bad and we will be moving away from using bools to specify and using enums to control

Reply:
Thank you for your patient review, have updated!